### DISCUSSION TITLE
丰富restApi的功能

### DISCUSSION THREAD

**Discussion by dc-vin** [hhyo/Archery#2883] [Ideas]
希望能根据工单id或者工单工单名称查询工单的详细信息，包括执行时间、执行状态等。现在通过列表查询满足不了需求，我在自己的系统里面通过rest api提交工单，执行统一通过 Archery执行，执行完成后我无法在自己的系统通过api查询执行状态

**Comment by nick2wang** [comment]
工单列表接口是支持过滤的
get /api/v1/workflow/?id=123&workflow__workflow_name__icontains=工单名称

**Reply by dc-vin** [reply]
好的谢谢，我试试看

**Reply by dc-vin** [reply]
为什么文档中只有一个接口

![image](https://github.com/user-attachments/assets/7df02ff2-960d-4560-9f07-4739d5a81cb9)

**Reply by dc-vin** [reply]
参数只有两个，版本 1.9.1
